src/components/app.js

```diff
@@ -167,7 +169,13 @@ const App = () => {
         setShowParameters(false);
     }
 
-    const config = configs.length > 0 ? configs[tabIndex] : null;
+    function toggleTab(newTabValue) {
+        history.push(newTabValue);
+    }
+
+    const mapProcessToIndex = new Map(
```

Comment:
You don't need this. A map with a small number of entry is not more efficient than a array search.

Reply:
Map is no more used.